README.md

```diff
@@ -1,2 +1,138 @@
+- React Router: Library for routing and navigation in a React application.
+- Axios: Promise-based HTTP client for making API requests.
+- Leaflet: JavaScript library for interactive maps.
+- Jest and React Testing Library: Testing frameworks for unit and integration testing.
+- CSS Modules: CSS styling approach that scopes styles to individual components.
+- Responsive Design: CSS media queries and responsive layout techniques for optimal viewing across device
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Park Listings: Browse a list of national parks withing all the states of the U.S.
+- Park Details: View detailed information about each park, including descriptions, total activities and topics.
+- Search Functionality: Search for parks based on the state code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://wild-vista.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To begin, clone the repository and install all dependencies by running `npm install`. Once the installation is complete, run the project using `npm start`.
+
+### Prerequisites
+
+In order to run this project you need node.js
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Cristian "ZamCham"**
+
+- GitHub: [@zam-cham](https://github.com/zam-cham)
+- Twitter: [@zamchamgames](https://twitter.com/zamchamgames)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/cristian-zamcham)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Filtering and Sorting: Filter and sort the park list based on various criteria such as location, activities, and park name.
+- Interactive Map: Explore parks visually on an interactive map, allowing users to discover parks in specific regions.
+- Favorites: Save parks as favorites for quick access and future reference.
+- Responsive Design: The application is designed to be responsive and optimized for various screen sizes, including desktop, tablet, and mobile devices.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, let us know on any of my social media account ;)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse team
+
```

Comment:
[REQUIRED]As stated in the requirement and also mentioned by the previous reviewer, kindly acknowledge  [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the "Acknowledgements" section. 🥺 🙏